wordpress.org/public_html/wp-content/plugins/official-wordpress-events/template-events.php

```diff
@@ -9,20 +9,26 @@
 		<ul class="ofe-event-list">
 			<?php foreach ( $day_events as $event ) : ?>
 				<li>
-					<?php if ( $event->location ) : ?>
-						<?php echo esc_html( 'online' === $event->location ? 'Online' : $event->location ); ?>
-						<span class="owe-separator"></span>
+					<?php // WordCamp.org doesn't collect start times, so only display those for meetups ?>
+						<?php if ( 'wordcamp' != $event->type ) : ?>
+							<?php echo date( 'g:i a', $event->start_timestamp ); ?>
+							<br>
 					<?php endif; ?>
 
 					<a href="<?php echo esc_url( $event->url ); ?>">
 						<?php echo esc_html( $event->title ); ?>
+						<br>
 					</a>
-
-					<?php // WordCamp.org doesn't collect start times, so only display those for meetups ?>
-					<?php if ( 'wordcamp' != $event->type ) : ?>
+					
+					<?php if ( $event->type ) : ?>
+						<?php echo esc_html( mb_convert_case( $event->type, MB_CASE_TITLE, "UTF-8") ); ?>
```

Comment:
`WordCamp` is technically spelled with a capital `C`, so we may need something here that converts the types to hardcoded labels.